.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/graphql"]
+	path = src/graphql
+	url = https://github.com/otterize/api
```

Comment:
same comment here re using the graphql API defined in otterize/graphql (it's different, and we'll keep only that)

Reply:
done :)